pages/common/flutter.md

```diff
@@ -0,0 +1,23 @@
+# flutter
+
+> Google’s free and open source cross-platform mobile app SDK.
```

Comment:
Perhaps ...`free, open source, and cross-platform`.... would read better?

Reply:
That's right, that sounds better.